app/src/main/res/values/shape.xml

```diff
@@ -47,4 +47,12 @@
     <item name="cornerRadius">2dp</item>
   </style>
 
+  <style name="ShapeAppearanceOverlay.Simple.AddNewCustomDrugButton" parent="ShapeAppearance.MaterialComponents.MediumComponent">
```

Comment:
Just an FYI, we don't have to extend the parents for overlays. Whether it might be for `ThemeOverlays` or `ShapeAppearanceOverlay` we simple extend an empty parent.

Reply:
I was following the pattern that was followed in the above Shape Appearance styles.

Reply:
Those were shape appearance styles, here we are creating an overlay. You can take a look at `ShapeAppearanceOverlay.Simple.BottomSheet` for example.

Reply:
One of the reasons why this can cause issues is, it will inherit fields that we are not overriding from the parent. Instead what a overlay should do is, override from our theme shape appearances.

Reply:
@vinaysshenoy give me a moment. Pushing this change as well.

Reply:
@vinaysshenoy done